src/standalone.js

```diff
@@ -1,13 +1,13 @@
 import * as Babel from "@babel/standalone";
 
 // take from @babel/standalone
-import {availablePlugins, availablePresets} from '@babel/standalone';
+import { availablePlugins, availablePresets } from '@babel/standalone';
 
 export function transpilePlugin(pluginString, presets) {
   return Babel.transform(pluginString, {
     babelrc: false,
     configFile: false,
-    ast: false,
+    ast: true,
```

Comment:
What does this option do?

Reply:
From testing, it doesn't seem to do anything different in terms of this PR's feature